modules/alloy/src/main/ts/ephox/alloy/ui/schema/MenuSchema.ts

```diff
@@ -42,6 +42,7 @@ const configureMatrix = (detail: MenuDetail, movementInfo: MenuMatrixMovement):
     row: movementInfo.rowSelector,
     cell: '.' + detail.markers.item
   },
+  previousSelector: movementInfo.previousSelector,
```

Comment:
Ideally, we'd have a test for this in alloy....

Reply:
I shall add a test
[ ] Add a test